core/domain/acl_decorators.py

```diff
@@ -989,6 +989,26 @@ def test_can_create_topic(self, **kwargs):
     return test_can_create_topic
 
 
+def can_access_topics_and_skills_dashboard(handler):
+    """Decorator to check whether the user can create a topic."""
```

Comment:
This docstring is not correct.

In general docstrings should have a section for args etc. as well. @apb7 isn't this supposed to be caught by the current linter? /cc @kevinlee12

Reply:
@seanlip: We have an option to make all sections of the docstrings mandatory. By default, it is not enabled. Since we have some docstrings without all sections, this option was not enabled yet.

Reply:
Ah ok, thanks. Is this issue tracked on the issue tracker somewhere? If not we should do so (something along the lines of "update docstrings A, B, C, ..., Y, then turn the flag on".

Reply:
Done, changed the docstring.